### ISSUE TITLE
Proposal for a gallery for UI components

### ISSUE THREAD

**Issue by 0xDevansh** [quasarframework/quasar#8404]
**Is your feature request related to a problem? Please describe.**
While browsing through what Quasar has to offer in UI components, the long list makes me feel a bit lost. If one is searching for a particular component he/she has in mind, but doesn't know what it's called, it can be a hard task clicking through the items at the [component page](https://quasar.dev/vue-components). This can be further aggravated for people with slower internet speed.

**Describe the solution you'd like**
My suggestion: Having a 'gallery' containing samples of all components in a single page, similar to Bootstrap's [cheat sheet](https://getbootstrap.com/docs/5.0/examples/cheatsheet/). This would be a faster and easier way to browse through the offered components, and a link to the component's documentation page can also be attached for further reading.

**Comment by DeathVenom54** [comment]
New open source contributor here, I would love to work on this!

**Comment by hawkeye64** [comment]
@DeathVenom54 I was working on a MadeWithQuasar site. I ran into issues and stopped. Also, I wanted to wait for Quasar v2.
It's almost time to pick it up again. Would love to have others helping.

**Comment by IlCallo** [comment]
I think he's talking more of a page like https://material.angular.io/components/categories
We may add this, as we need to change a bit docs structure in some time

**Comment by DeathVenom54** [comment]
> I think he's talking more of a page like https://material.angular.io/components/categories
> We may add this, as we need to change a bit docs structure in some time

Exactly. I tried messing with the docs repo code and a lot doesn't make sense to me, so I'll leave it to the elites 😅

**Comment by yusufkandemir** [comment]
We will have a page for this in the incoming Quasar Docs Website GUI Update (#13177).

Here is how it looks like:
![image](https://user-images.githubusercontent.com/6266078/193447928-19c56398-6fff-4dcb-b4b6-263a42e76033.png)

**Comment by rstoenescu** [comment]
This landed a long time ago: https://quasar.dev/components

It is a single page listing every component, directive, plugin, composable and util as a card with a preview image, with free-text search and category filters (Buttons, Inputs, Loading, Media, Navigation, Panels & Popups, Scroll, Tables, ...), so you can browse by what a thing looks like or does rather than by its name. The old `/vue-components`, `/vue-directives`, `/quasar-plugins`, `/vue-composables` and `/quasar-utils` listing URLs all redirect there.

Thanks for the proposal!
